cli/007-cli-color.md

```diff
@@ -0,0 +1,13 @@
+- Start Date: 2021-06-11
+- RFC PR:
+- Issue:
+
+### Better coloring
+
+Good usage of coloring can help users to read the output and direct attention to the most interesting parts.
+
+- Standartize the colors used in printing
+- Work through commands and improve the coloring
+- add a flag `--no-color` to disable coloring alltogether
```

Comment:
We could have `--color/--no-color` with color being the default for TTY and no color for non-TTY output.
You can override the default via the proposed configuration mechanism.

Reply:
yea better control over enabling and disabling coloring in tty/non-tty makes sense
 
i've noticed other modern CLIs achieve this by providing the following single flag with values:

- nushell's `ls --help` gives the following:
      -c, --color <when>                 When to use colors: never, *auto*, always

- same with `bat --help`:
            --color <when>                When to use colors (*auto*, never, always).
            
this kinda makes sense since this is just one flag (doesn't pollute help with many flags) and allows to switch between three modes: auto that checks for tty, always or never coloring

Reply:
Yeah, that's a really good suggestion, it's been standard in unix CLI tools for a while now, `grep` is another example.

Reply:
```suggestion
- add a flag `--color` that takes the following values: `auto` (enable in tty/ disable in non-tty), `always`, `disabled` 
```